organization-bootstrap/environments/main.tf

```diff
@@ -141,6 +141,9 @@ module "log-sink-audit" {
 ###############################################################################
 
 # shared resources project
+# you might want to create a 'Shared' folder if you need multiple projects for
```

Comment:
Could we be prescriptive and default to this? It's not a big harm IMO.

Reply:
I would prefer keeping this as simple as possible, and having it match he customer installs we did in the past. Is this something you strongly feel for?

Reply:
I'd strongly prefer to have consistency. This 90% matches how we do things typically already, so I think it's valuable to just use it as a default.

Reply:
Ok, lemme then make that change (I also need to change the diagram), and switch to using comment for the simpler setup.

Reply:
After giving it a bit of thought, if a Shared folder is needed, and shared resources go beyond a simple project, then the best way to implement it is by adding a "shared" item to the environments list, and treating shared resources as one extra environment (with their own service account, bucket, Terraform setup, and pipeline).

If that is not needed (and it never was in all our automation engagements), a simple top-level project managed in the bootstrap stage is best, typically providing a simple way to manage GCR, KMS and a handful of buckets.

So I'm not adding an extra folder, but I'm detailing the above in the README.